onnxruntime/core/providers/webnn/builders/impl/mha_op_builder.cc

```diff
@@ -0,0 +1,369 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Intel Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/shared/utils/utils.h"
+#include "core/providers/webnn/builders/helper.h"
+#include "core/providers/webnn/builders/model_builder.h"
+#include "core/providers/webnn/builders/op_builder_factory.h"
+#include "cmath"
```

Comment:
```suggestion
#include <cmath>
```
System header includes should use global `<>` rather than local `""`.